lib/conditions/predefined.js

```diff
@@ -154,5 +154,13 @@ module.exports = [
     schema: {
       $id: 'http://express-gateway.io/schemas/conditions/anonymous.json'
     }
+  },
+  {
+    name: 'clientAuthorized',
```

Comment:
1) what is `clientAuthorized`? Can it have a more descriptive name? It is confusing with auth policies we have. 
2) it is really common use case ? why it can't be own module?

Reply:
I'm fine with both

1. A better descriptive name (I was just replicating the logic `client.authorized`
2. Have it as a separate module. @altsang is that ok with you?